content/en/docs/tasks/security/proxy-cert/index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Provision a certificate through a sidecar [Experimental]
+description: Provision a certificate through a sidecar.
+weight: 100
+keywords: [certificate,sidecar]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+Istio sidecars can obtain certificates through
+the secret discovery service.
+For an application without a sidecar to get a certificate,
+it may deploy a sidecar to generate a certificate through
+the secret discovery service and share the certificate with the application
+through a shared memory.
+This task uses Prometheus as an example application to show provisioning
+a certificate through such mechanism.
+
+## Getting started
+
+Install Istio:
+
+{{< text bash >}}
+$ istioctl manifest apply
```

Comment:
You are giving an example here. I think it's more common to use certificate when the mTLS is enabled?
Can you copy the following to be consistent with other tasks?

* Install Istio with the `global.mtls.enabled` option set to false and `global.mtls.auto` set to true.
For example, using the `demo` configuration profile:

{{< text bash >}}
$ istioctl manifest apply --set profile=demo \
  --set values.global.mtls.auto=true \
  --set values.global.mtls.enabled=false
{{< /text >}}

Reply:
Done.